src/store/modules/order/actions.ts

```diff
@@ -76,7 +76,15 @@ const actions: ActionTree<OrderState , RootState> ={
   },
 
   updateCurrent ({ commit }, payload) {
-    commit(types.ORDER_CURRENT_UPDATED, { order: payload.order })
+    const order = {
+      ...payload.order,
+      firstName: payload.order.customerName?.split(' ').slice(0, -1).join(" "),
```

Comment:
I think we should first check if customer has string, split it, store it into a variable and then assign to first name & last name